macos/FluentUIResources-macos/FluentUI.xcassets/ButtonView/Contents.json

```diff
@@ -0,0 +1,9 @@
+{
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  },
+  "properties" : {
+    "provides-namespace" : true
```

Comment:
Should the name of this namespace just be "Button" instead of "ButtonView" to match the control name?

Reply:
We run into the same issue in FURN and I'd like to be consistent with naming whatever we decide. Personally I prefer dropping the word "view" as it better aligns with Apple control names and this is an Apple repo